docs/docs/user-guide/cmdlets/Update-M365DSCAzureAdApplication.md

```diff
@@ -63,4 +84,8 @@ This function does not generate any output.
 
 `Update-M365DSCAzureAdApplication -ApplicationName 'Microsoft365DSC' -Permissions @(@{Api='SharePoint';PermissionName='Sites.FullControl.All'},@{Api='Graph';PermissionName='Group.ReadWrite.All'},@{Api='Exchange';PermissionName='Exchange.ManageAsApp'}) -AdminConsent -Credential $creds -Type Certificate -CertificatePath c:\Temp\M365DSC.cer`
 
+-------------------------- EXAMPLE 4 --------------------------
+
+`Update-M365DSCAzureAdApplication -ApplicationName $Microsoft365DSC -Permissions $(Get-M365DSCCompiledPermissionList -ResourceNameList Get-M365DSCAllResources -PermissionType Application -AccessType Read) -Type Certificate -CreateSelfSignedCertificate -AdminConsent -MonthsValid 12 -Credential $creds -CertificatePath c:\Temp\M365DSC.cer`
```

Comment:
```suggestion
This is the easiest way to create a new app registration for Microsoft365DSC in a tenant.
It will connect to the tenant with the credentials **$creds** (important to identify which tenant to connect to), create/update the Microsoft365DSC application named "Microsoft365DSC", give it Read access to all possible resources, generate a self-signed certificate valid for max possible duration (12 months - default value, save it to a file c:\Temp\M365DSC.cer and upload this cert into the newly created app registration. 

`Update-M365DSCAzureAdApplication `
    -ApplicationName Microsoft365DSC `
    -Permissions $(Get-M365DSCCompiledPermissionList -ResourceNameList Get-M365DSCAllResources -PermissionType Application -AccessType Read) `
    -AdminConsent `
    -Type Certificate `
    -CreateSelfSignedCertificate `
    -CertificatePath c:\Temp\M365DSC.cer `
    -Credential $creds
    
```